### ISSUE TITLE
Exceptions during test loading cause a test runner exception

### ISSUE THREAD

**Issue by dwalleck** [CafeHub/opencafe#66]
Expected behavior: If an exception occurs when trying to load a test repository, it should be logged but the runner should continue

Actual behavior: An exception occurs when trying to log the error due to a bug with logging. The exception is treated as a string, which causes another exception at the runner level:

File "/home/xxxxxxx/.virtualenvs/cafevirt/bin/cafe-parallel", line 11, in <module>
    sys.exit(entry_point())
  File "/home/xxxxxxx/.virtualenvs/cafevirt/local/lib/python2.7/site-packages/cafe/drivers/unittest/runner_parallel.py", line 283, in entry_point
    runner = UnittestRunner()
  File "/home/xxxxxxx/.virtualenvs/cafevirt/local/lib/python2.7/site-packages/cafe/drivers/unittest/runner_parallel.py", line 80, in __init__
    self.cl_args.testrepos = import_repos(self.cl_args.testrepos)
  File "/home/xxxxxxx/.virtualenvs/cafevirt/local/lib/python2.7/site-packages/cafe/drivers/unittest/runner_parallel.py", line 61, in import_repos
    "Runner", "import_repos", repo_name, exception)
  File "/home/xxxxxxx/.virtualenvs/cafevirt/local/lib/python2.7/site-packages/cafe/drivers/base.py", line 208, in print_exception
    print("{0}:".format(safe_decode(exception)), file=sys.stderr, end=" ")
  File "/home/xxxxxxx/.virtualenvs/cafevirt/local/lib/python2.7/site-packages/cafe/common/decoding.py", line 17, in safe_decode
    return text.decode(incoming, errors)
AttributeError: 'exceptions.TypeError' object has no attribute 'decode'

The print_exception function should pass the name of the exception and not the object itself as the exception name.

**Comment by dougPhilips** [comment]
I tried this out by pulling the latest master and deliberately introducing an import of a nonexistant module and then removing a colon after an 'if' expression. Both provoked this stack trace:
```
======================================================================
Suite Builder: import_module: qmroast.ui.raptor.core_functionality.smoke: Traceback (most recent call last):
  File "/Users/doug9425/.virtual_envs/6d76fb52637a742/bin/cafe-parallel", line 11, in <module>
    load_entry_point('opencafe==0.3.4', 'console_scripts', 'cafe-parallel')()
  File "/Users/doug9425/.virtual_envs/6d76fb52637a742/lib/python2.7/site-packages/cafe/drivers/unittest/runner_parallel.py", line 283, in entry_point
    runner = UnittestRunner()
  File "/Users/doug9425/.virtual_envs/6d76fb52637a742/lib/python2.7/site-packages/cafe/drivers/unittest/runner_parallel.py", line 89, in __init__
    exit_on_error=self.cl_args.exit_on_error).get_suites()
  File "/Users/doug9425/.virtual_envs/6d76fb52637a742/lib/python2.7/site-packages/cafe/drivers/unittest/suite_builder.py", line 44, in get_suites
    for class_ in self._get_classes(self._get_modules()):
  File "/Users/doug9425/.virtual_envs/6d76fb52637a742/lib/python2.7/site-packages/cafe/drivers/unittest/suite_builder.py", line 96, in _get_modules
    exception)
  File "/Users/doug9425/.virtual_envs/6d76fb52637a742/lib/python2.7/site-packages/cafe/drivers/base.py", line 208, in print_exception
    print("{0}:".format(safe_decode(exception.__name__)), file=sys.stderr, end=" ")
AttributeError: 'exceptions.ImportError' object has no attribute '__name__'
```

This was with Python 2.7.14 on High Sierra, with the python installed with brew.

**Comment by dwalleck** [comment]
I made a silly mistake and fixed the issue in #69. Going to circle back with folks to make sure this is resolved

**Comment by dwalleck** [comment]
Resolved in PR #69.
